rdk/template/terraform/1.x/config_rule.tf

```diff
@@ -0,0 +1,201 @@
+locals {
+  event_triggered        = (length(var.source_events) > 0 ? true : false)
+  periodic_triggered     = var.source_periodic != "NONE" ? true : false
+  create_new_lambda_role = (var.lambda_role_arn == "NONE" ? true : false)
```

Comment:
Is the string literal "NONE" something that the user will need to provide as a value? I'm confused how this comparison works.